### DISCUSSION TITLE
Better collision detection - 2D Physics

### DISCUSSION THREAD

**Discussion by GorEldeen** [godotengine/godot-proposals#7175] [2D]
Hello dear godot team, I am not a physics professional or anything but i saw somebody on youtube that discusses how to make collision detection in 2d physics better and much more efficient

Here's the video i'm talking about: https://www.youtube.com/watch?v=9IULfQH7E90&ab_channel=Pezzza%27sWork

Do you think you can do the same thing for godot 2d physics?

**Comment by AThousandShips** [comment]
* See: #3704

**Comment by AThousandShips** [comment]
Also:
* Duplicate of: #6610

**Reply by GorEldeen** [reply]
thank you sir

**Reply by AThousandShips** [reply]
Not a sir, please be attentive to my profile before using pronouns or honorifics

**Reply by GorEldeen** [reply]
I apologise, i am a bit new to this place
